### PR TITLE
Updating JQuery to 1.7.2 and Adding JQuery UI

### PR DESCRIPTION
**Pull request by fism88**
Upgraded to JQuery 1.7.2 and added JQuery UI to public/javascripts

### CODE REVIEW

**Comment by amshali** [comment]
Let me review this. Do we need all the 425 files?

**Comment by fism88** [comment]
I'm not sure, the website had a whole customized package of the UI to download so I gave that a try. So that's why it has all the extra files probably.

**Comment by ProCynic** [comment]
You can use the version of jquery that comes with bootstrap.

`<script src="/public/bootstrap/docs/assets/js/jquery.js"></script>`

File: app/controllers/Profiles.java.orig [hunk 1]
```diff
@@ -0,0 +1,172 @@
+package controllers;
```
Comment (amshali) [inline app/controllers/Profiles.java.orig]: What is this file? Remove please. Make sure you pull before push.

File: app/controllers/Application.java [hunk 2]
```diff
@@ -167,6 +167,7 @@ public static void account_save(User update, String old_password) {
   public static void edit_basic(Long userID) {
     User user = User.findById(userID);
 	Profile profile = Profile.find("owner = ?", user).first();
+	  user.profile.relationshipStatus = "Married";
```
Comment (amshali) [inline app/controllers/Application.java]: What? Married? Why?

File: app/controllers/Profiles.java.orig (hunk 1 above)
Comment (fism88) [inline app/controllers/Profiles.java.orig]: sorry, this is the aftermath of merging and pulling. I removed it

File: app/controllers/Application.java (hunk 2 above)
Comment (fism88) [inline app/controllers/Application.java]: Oh I was testing for updating the Anniversary. I took it out.

**Comment by fism88** [comment]
So should I use the jquery version that bootstrap provides in main.html for the whole application to use or just use it for edit_basic.html that is having the problem?

**Comment by amshali** [comment]
I think for the whole app.

**Comment by amshali** [comment]
Make sure you test it before committing. Thanks.

**Comment by ProCynic** [comment]
what I did in the bootstrap version of the site was just replace the script that imports jquery in the main page with an import of the bootstrap version of jquery.

**Comment by fism88** [comment]
okay I'll do that. It's just a difference of jquery 1.7.1 from bootstrap vs 1.7.2 from jquery ui, but I haven't seen any problems because of it yet. I'm going to keep testing first before I commit it, though.

**Comment by amshali** [comment]
So the jquery bootstrap version is 1.7.2?

**Comment by fism88** [comment]
No, the JQuery UI version is 1.7.2. bootstrap is 1.71. Geoffrey, correct me if I'm mistaken

**Comment by ProCynic** [comment]
yeah, bootstrap is v1.7.1.  If that's sufficient I don't see any reason to
include another as well, but if it isn't go ahead and replace it.
-Geoffrey Parker

On Sun, Apr 22, 2012 at 5:21 PM, Faraaz Ismail <
reply@reply.github.com

**Comment by amshali** [comment]
I think it might be sufficient.

**Comment by amshali** [comment]
I am gonna go ahead and merge this, but fix this at your convenience.

**Pull request merged**